tests/e2e/specs/customizer/blog/single-post/content-width.test.js

```diff
@@ -0,0 +1,40 @@
+import { createURL, createNewPost, publishPost } from '@wordpress/e2e-test-utils';
+import { setCustomize } from '../../../../utils/customize';
+describe( 'Single post in the customizer', () => {
+	it( 'content width default should apply corectly', async () => {
+		const contentWidth = {
+			'blog-single-width': 'default',
+			'blog-single-max-width': 1200,
+		};
+		await setCustomize( contentWidth );
+		await createNewPost( { postType: 'post', title: 'hello world' } );
+		await publishPost();
+		await page.goto( createURL( '/hello-world/' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '.single-post .site-content > .ast-container' );
```

Comment:
Could you please verify the selector:- https://share.bsf.io/X6uEO6xK
```suggestion
		await page.waitForSelector( '..ast-container' );
```

Reply:
I checked selector at my end it is correct --> https://share.getcloudapp.com/5zuLXXB7